src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -204,6 +204,85 @@
     } while (0)
 #endif
 
+MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_need_request_creation(const MPIR_Request * req)
```

Comment:
I'm pretty confused by this patch as a whole. This needs some better explanation.